src/components/TeacherComponents/helpers/downloadAndShareHelpers/getFormattedDetailsForTracking.tsx

```diff
@@ -48,8 +48,8 @@ export const getSchoolName = (
 export const extractUrnAndSchool = (school: string) => {
   const match = /^(?:(\d{7}|\d{6}|\d{3}-\d{4}))-(.*)/.exec(school);
   return {
-    urn: match ? match[1] : "",
-    schoolName: match ? match[2] : "",
+    urn: match?.at(1),
+    schoolName: match?.at(2),
```

Comment:
Returning an empty string when the Regex failed seemed a bit counter intuitive — especially when the type was already `{ url: string | undefined; schoolName: string | undefined }`. A consumer would reasonably expect the presence of a string to mean that the values had been extracted successfully.